gallery/src/components/NxDropdown/NxDropdownRightButtonsExample.tsx

```diff
@@ -14,30 +14,32 @@ function NxDropdownRightButtonsExample() {
 
   return (
     <NxDropdown label="Navigation" isOpen={isOpen} onToggleCollapse={onToggleCollapse}>
-      <a href="#/" className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button" role="menuitem">
         <span className="nx-dropdown-button-content">Text Link</span>
       </a>
-      <a href="#/" className="nx-dropdown-button">
+      <a href="#/" className="nx-dropdown-button" role="menuitem">
         <span className="nx-dropdown-button-content">
           Text Link - this link should trigger truncation
         </span>
       </a>
       <NxButton onClick={() => alert('icon click')}
                 className="nx-dropdown-right-button"
                 variant="icon-only"
-                title="Delete Button Link2">
+                title="Delete Button Link2"
+                role="menuitem">
```

Comment:
Check with design about how these buttons should be added into the recommended patterns.  These are pretty nonstandard, so I think we're on our own in terms of how they behave.  Having them just be part of the up/down key cycling doesn't necessarily seem right though.